package.json

```diff
@@ -82,6 +123,25 @@
 					"when": "editorTextFocus",
 					"group": "LaunchDarkly"
 				}
+			],
+			"view/title": [
+				{
+					"command": "ldFeatureFlags.refreshEntry",
+					"when": "view == ldFeatureFlags",
+					"group": "navigation"
+				}
+			],
+			"view/item/context": [
+				{
+					"command": "ldFeatureFlags.copyKey",
+					"when": "view == ldFeatureFlags && viewItem == flagViewKey",
+					"group": "inline"
+				},
+				{
+					"command": "ldFeatureFlags.openBrowser",
+					"when": "view == ldFeatureFlags && viewItem == flagViewBrowser",
+					"group": "inline"
+				}
```

Comment:
Can you add a short note on what each of these new menus do - are these necessary for the explorer behavior?